src/script/components/Conversation/ConversationCells/CellsTable/CellsTable.tsx

```diff
@@ -183,6 +178,9 @@ export const CellsTable = ({
                     key={cell.id}
                     css={cell.column.id === 'id' ? tableActionsCellStyles : tableCellStyles}
                     data-cell={cell.column.id === 'id' ? undefined : cell.column.columnDef.header}
+                    style={{
+                      width: cell.column.id == 'name' ? undefined : cell.column.getSize(),
+                    }}
```

Comment:
Minor suggestion, if you want to do it:
```suggestion
               {...(cell.column.id == 'name' && {
                style: {
                   width: cell.column.getSize(),
                },
              })}
```

Reply:
I think it complicates the whole thing

Reply:
Not a fan of this approach as well. It's less readable, and it also prints "false" for the falsy case